Unable to build `policy` when `DISTRO = debian`: tmp/base.fc.tmp:1166: unable to parse a file context line

## Issue
When setting `DISTRO = debian` you may encounter issues regarding syntax errors when parsing `tmp/base.fc.tmp`
 file:

**Error**:
```
$> make policy
Compiling test base module
/usr/bin/checkmodule base.conf -o tmp/base.mod
Creating test base module file contexts.
m4 -D distro_debian -D enable_ubac -D mls_num_sens=16 -D mls_num_cats=1024 -D mcs_num_cats=1024 -D hide_broken_symptoms support/divert.m4 policy/support/file_patterns.spt policy/support/ipc_patterns.spt policy/support/loadable_module.spt policy/support/misc_macros.spt policy/support/misc_patterns.spt policy/support/mls_mcs_macros.spt policy/support/obj_perm_sets.spt support/undivert.m4 tmp/generated_definitions.conf policy/modules/kernel/corecommands.fc policy/modules/kernel/corenetwork.fc policy/modules/kernel/devices.fc policy/modules/kernel/domain.fc policy/modules/kernel/files.fc policy/modules/kernel/filesystem.fc policy/modules/kernel/kernel.fc policy/modules/kernel/mcs.fc policy/modules/kernel/mls.fc policy/modules/kernel/selinux.fc policy/modules/kernel/storage.fc policy/modules/kernel/terminal.fc policy/modules/kernel/ubac.fc > tmp/base.fc.tmp
python3 -E support/fc_sort.py tmp/base.fc.tmp base.fc
tmp/base.fc.tmp:1166: unable to parse a file context line: '
make: *** [Rules.modular:183: base.fc] Error 1
```
This works fine when setting to `redhat`.

## Config
**build.conf**:
```
########################################
#
# Policy build options
#

# Policy version
# By default, checkpolicy will create the highest
# version policy it supports.  Setting this will
# override the version.  This only has an
# effect for monolithic policies.
#OUTPUT_POLICY = 18

# Policy Type
# standard, mls, mcs
TYPE = standard

# Policy Name
# If set, this will be used as the policy
# name.  Otherwise the policy type will be
# used for the name.
NAME = targeted-gyptazy

# Distribution
# Some distributions have portions of policy
# for programs or configurations specific to the
# distribution.  Setting this will enable options
# for the distribution.
# redhat, gentoo, debian, suse, and rhel4 are current options.
# Fedora users should enable redhat.
DISTRO = debian

# Unknown Permissions Handling
# The behavior for handling permissions defined in the
# kernel but missing from the policy.  The permissions
# can either be allowed, denied, or the policy loading
# can be rejected.
# allow, deny, and reject are current options.
#UNK_PERMS = deny

# Direct admin init
# Setting this will allow sysadm to directly
# run init scripts, instead of requring run_init.
# This is a build option, as role transitions do
# not work in conditional policy.
DIRECT_INITRC = n

# Build monolithic policy.  Putting y here
# will build a monolithic policy.
MONOLITHIC = n

# User-based access control (UBAC)
# Enable UBAC for role separations.
UBAC = y

# Custom build options.  This field enables custom
# build options.  Putting foo here will enable
# build option blocks named foo.  Options should be
# separated by spaces.
CUSTOM_BUILDOPT =

# Number of MLS Sensitivities
# The sensitivities will be s0 to s(MLS_SENS-1).
# Dominance will be in increasing numerical order
# with s0 being lowest.
MLS_SENS = 16

# Number of MLS Categories
# The categories will be c0 to c(MLS_CATS-1).
MLS_CATS = 1024

# Number of MCS Categories
# The categories will be c0 to c(MLS_CATS-1).
MCS_CATS = 1024

# Set this to y to only display status messages
# during build.
QUIET = n
```

## Output file
`tmp/base.fc.tmp`:
You can see the single quote in L6 that raises this issue:
```
# this is a static /dev dir "backup mount"
# if you want to disable udev, youll have to boot permissive and relabel!
/dev/\.static           -d      system_u:object_r:device_t
/dev/\.static/dev       -d      system_u:object_r:device_t
/dev/\.static/dev/(.*)?         <<none>>
'

/etc/udev/devices       -d      system_u:object_r:device_t

# used by init scripts to initally populate udev /dev
/lib/udev/devices(/.*)?         system_u:object_r:device_t
/lib/udev/devices/lp.*  -c      system_u:object_r:printer_device_t
/lib/udev/devices/null  -c      system_u:object_r:null_device_t
/lib/udev/devices/zero  -c      system_u:object_r:zero_device_t
```

Fixable in two ways by adjusting the parsing in a more complex way (for the comment include) or (it only occurs a single time) in `policy/modules/kernel/devices.fc` in `ifdef(distro_debian)`.

## How to Reproduce
1. Set `DISTRO = debian` in `build.conf`
2. `make clean`
3. `make conf`
4. `make policy`

--- a/policy/modules/kernel/devices.fc
+++ b/policy/modules/kernel/devices.fc
@@ -245,7 +245,7 @@ ifdef(`distro_suse', `
 
 ifdef(`distro_debian',`
 # this is a static /dev dir "backup mount"
-# if you want to disable udev, you'll have to boot permissive and relabel!
+# if you want to disable udev, you will have to boot permissive and relabel!
 /dev/\.static		-d	gen_context(system_u:object_r:device_t,s0)
 /dev/\.static/dev	-d	gen_context(system_u:object_r:device_t,s0)
 /dev/\.static/dev/(.*)?		<<none>>

